src/webgpu/web_platform/copyToTexture/ImageBitmap.spec.ts

```diff
@@ -180,7 +204,10 @@ g.test('from_ImageData')
       orientation === 'flipY'
     );
 
-    const format: RegularTextureFormat = t.formatForExpectedPixels(dstColorFormat);
+    const format: RegularTextureFormat =
+      kTextureFormatInfo[dstColorFormat].baseFormat !== undefined
+        ? kTextureFormatInfo[dstColorFormat].baseFormat!
+        : dstColorFormat;
```

Comment:
nit: can use `kTextureFormatInfo[dstColorFormat].baseFormat ?? dstColorFormat` instead of `!`.